include/wlr/types/wlr_screencopy_v1.h

```diff
@@ -27,9 +27,14 @@ struct wlr_screencopy_manager_v1 {
 	void *data;
 };
 
+struct wlr_screencopy_connection_v1 {
```

Comment:
Rename to `wlr_screencopy_v1_client` for consistency with other places in wlroots where we do this.